modules/loader-utils/src/lib/worker-utils/worker-pool.js

```diff
@@ -27,15 +27,15 @@ export default class WorkerPool {
    * Process binary data in a worker
    */
   async process(data, jobName) {
-    return new Promise((resolve, reject) => {
+    return await new Promise((resolve, reject) => {
```

Comment:
Just remove `async` and return the Promise.

async/await is transpiled with additional overhead. Using the syntax does not provide any additional clarity to this code.